core/templates/pages/topic-editor-page/topic-editor-page.component.html

```diff
@@ -17,7 +17,7 @@
              [ngClass]="{'navbar-tab-active-icon': isMainEditorTabSelected() }">
           </i>
         </a>
-        <div ngbDropdownToggle
+        <div ngbDropdownAnchor
```

Comment:
@heyimShivam FYI, this issue got introduced through PR #17372 . Adding you to this thread to inform the migration team about this issue (as I'm worried replacing uib-dropdown with ngbDropdown from other places will lead to similar problems.).

I've used [ngbDropdownAnchor](https://ng-bootstrap.github.io/#/components/dropdown/api#NgbDropdownAnchor) to avoid having click events on the items which creates weird style issues on the warning container.

Also, I was wondering why are we using ngbDropdown for these cases, I think dropdowns are for a list of buttons and are written in a way to handle buttons. Should we use [popover](https://ng-bootstrap.github.io/#/components/popover/examples) instead to show warning messages?